packages/core/src/Card/Card.stories.tsx

```diff
@@ -74,6 +88,27 @@ export const BoxShadowsWithVaryingBorderRadii = () => (
   </Box>
 )
 
+export const Default = () => (
+  <a id='PABLO' href='www.priceline.com'>
+    Pablo
+  </a>
+)
+Default.parameters = {
+  pseudo: {rootSelector: "PABLO" },
+}
+
+export const Hover = () => (
+  <Box>
+  <StyledCard id='PABLO' boxShadowSize='sm'>
+    Card Text
+  </StyledCard>
+  </Box>
+)
```

Comment:
Can you try committing this file again without skipping the pre-commit hook? It looks like this is missing some formatting.

Reply:
strongly recommend using the `prettier` vscode extension + format on save. it will find our prettier config